src/Agent/NewRelic/Agent/Core/Metrics/MetricNameService.cs

```diff
@@ -30,18 +28,18 @@ public class MetricNameService : ConfigurationBasedService, IMetricNameService
 
         public string NormalizeUrl(string url)
         {
-            ITimer timer = new Timer();
```

Comment:
Side note: I'm not sure what this `timer` was doing in here in the first place.  It does seem safe to remove.